bigtable/client/mutations_test.cc

```diff
@@ -188,3 +188,22 @@ TEST(MutationsTest, MutipleRowMutations) {
   EXPECT_EQ("foo2", request.entries(0).row_key());
   EXPECT_EQ("foo3", request.entries(1).row_key());
 }
+
+/// @test Verify that variadic list of Mutations works as expected for
+/// SingleRowMutation.
+TEST(MutationsTest, SingleRowMutations) {
```

Comment:
Please add separate tests for taking a single variadic argument and two variadic arguments.

Reply:
added additional test case

Reply:
I do not see a test for two Mutations as arguments.  I explicitly asked about it because I think you might run into overload resolution problems with the constructor that takes an [iterator range](https://github.com/manish-qlogic/google-cloud-cpp/blob/966ffd28434dee68e222b63844c2f8ccdfe308e6/bigtable/client/mutations.h#L107).

If you do, feel free to remove the iterator range constructor, it was a bad idea.

Reply:
yes i ran into resolution problems, i have removed the iterator range constructor. 
The same situation in BulkMutation is handled by a special 2 argument constructor, I think it can also be removed too.